### PR TITLE
Make modules in other/ directory importable

### PR DESCRIPTION
**Pull request by larshum**
#21

### CODE REVIEW

**Review by mnil2 — CHANGES_REQUESTED** [review]
Good, but for one problem. Travis still reports one test failure in other. Specifically in thealgorithms.other.detecting_english_programmatically, it tries to load a dictionary or something that doesn't exist

Raw log:
```
[1m    def loadDictionary():[0m
[1m        path = os.path.split(os.path.realpath(__file__))[0m
[1m        englishWords = {}[0m
[1m>       with open(path[0] + '/Dictionary.txt') as dictionaryFile:[0m
[1m[31mE       FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/build/slarse/Python/thealgorithms/other/Dictionary.txt'[0m
```

File: thealgorithms/other/anagrams.py [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
+    with open(path[0] + '/words') as f:
+        word_list = sorted(list(set([word.strip().lower() for word in f])))
+
+    word_bysig = collections.defaultdict(list)
+    for word in word_list:
+        word_bysig[signature(word)].append(word)
+
+    print('finding anagrams...')
+    all_anagrams = {word: anagram(word)
+                    for word in word_list if len(anagram(word)) > 1}
```
Comment (mnil2) [inline thealgorithms/other/anagrams.py]: Actually, now that I look for it, you added args for anagram but call them here as in the original code.

**Review by mnil2 — APPROVED** [review]

**Comment by mnil2** [comment]
All good now :+1:

**Comment by larshum** [comment]
That other problem was because the "snake_case all things" commit changed the filename of the `dictionary.txt` to lowercase, but forgot to update it to lowercase in `detecting_english_programmatically.py`. I missed that error locally because my file system is case-insensitive. :laughing:

**Comment by mnil2** [comment]
I shouldve known it was the snake case guys fault all along

**Pull request merged**